guides/models/crecto/README.md

```diff
@@ -13,7 +13,7 @@ Add this to your application's `shard.yml`:
 ```yaml
 dependencies:
   crecto:
-    github: fridgerator/crecto
+    github: Crecto/crecto
```

Comment:
I think this should be lowercase

Reply:
This should probably be added to amber main as well. Perhaps it would fix the failing crecto specs.